Android/app/src/main/java/io/github/project_travel_mate/SplashActivity.java

```diff
@@ -46,4 +53,16 @@ protected void onCreate(Bundle savedInstanceState) {
             finish();
         }, 2000);
     }
+
+    /**
+     * Viewholder for the SplashActivity item
+     */
+    class ViewHolder {
```

Comment:
you removed viewholder from ChecklistActivity.java activity, but it's still here.
Can we have consistency in this refactoring?